src/towncrier/_builder.py

```diff
@@ -201,9 +208,10 @@ def issue_key(issue: str) -> tuple[int, str]:
         return (-1, issue)
 
 
-def entry_key(entry: tuple[str, Sequence[str]]) -> list[tuple[int, str]]:
-    _, issues = entry
-    return [issue_key(issue) for issue in issues]
+def entry_key(entry: tuple[str, Sequence[str]]) -> tuple[str, list[tuple[int, str]]]:
+    content, issues = entry
+    # Empty issues (unlinked fragments) should sort last by content.
```

Comment:
Not sure why... but I am not -1 for this :)

Reply:
👍🏻 It means we're consistent, and I updated the docs clarify that this is the behaviour.